pages/_app.js

```diff
@@ -25,6 +25,7 @@ function MyApp({ Component, pageProps }) {
         ) : (
           ""
         )}
+        <meta name="dcterms.service" content="ESDC-EDSC_SCLabs-Laboratoire" />
```

Comment:
@pcraig3 for the french name after the dash here, would it be ok to make it LabsSC? Adobe Analytics is fine with it but I mean as an abbreviation of the french name, do you think it makes sense?